preflight.sh

```diff
@@ -23,9 +23,9 @@ fi
 if [[ $PLATFORM == "gce" ]]; then
     # Set up network for running on gce
     echo "Setting up network for GCE"
-    bash rto_setup.sh
```

Comment:
Can we just check if have sudo and use sudo if we have it?

Reply:
Done.